src/quo/components/notifications/toast/style.cljs

```diff
@@ -38,8 +38,14 @@
   {:color (colors/theme-colors colors/white colors/neutral-100 theme)})
 
 (defn icon
-  [theme]
-  {:color           (colors/theme-colors colors/white colors/neutral-100 theme)
+  [type theme]
```

Comment:
Even though we are receiving a type, I'd suggested naming that var  differently (e.g. `icon-type` or whatever suits better). The reason is `type` is shadowing `clojure.core/type`.